### ISSUE TITLE
DESTDIR not heeded during make install

### ISSUE THREAD

**Issue by jengelh** [paul-j-lucas/cdecl#36]
First known bad version: cdecl-16.4
Last known good version: cdecl-16.3

## Issue Type

1. **Build-time issue**: you get either a compile-time error or warning while building **cdecl**.

## Build-time Issue

**What error or warning did you get?**

```
$ make install DESTDIR=/tmp/rt
...
make[3]: Entering directory '~/cdecl-16.4/share/bash-completions/completions'
make[4]: Entering directory '~/cdecl-16.4/share/bash-completions/completions'
make[4]: Nothing to be done for 'install-exec-am'.
mkdir -p /usr/local/share/bash-completions/completions
mkdir: cannot create directory ‘/usr/local/share/bash-completions’: Permission denied
make[4]: *** [Makefile:1551: /usr/local/share/bash-completions/completions] Error 1
```

**What compiler and version are you using?**

gcc version 13.2.1 20240509 [revision b7a2697733d19a093cbdd0e200ffce069a4bc812] (SUSE Linux)  amd64

**What operating system and version (Linux, FreeBSD, macOS, etc.) are you building on?**

Linux 6.9.0 amd64

**Comment by paul-j-lucas** [comment]
Try the HEAD revision and see if it works.

**Comment by paul-j-lucas** [comment]
Having not heard back and since I think the fix works as of 16.4.1, closing.

**Comment by jengelh** [comment]
works.
